ignore MSAPIModule validations when changing only the integration (#2825)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * Fixed an issue where **validate** failed with **is_valid_integration_file_path_in_folder** in cases of integrations with MSAPIModule.

Reviewer: ```suggestion
* Fixed an issue where **validate** failed with **is_valid_integration_file_path_in_folder** on integrations that use the `MSAPIModule`.
```
